TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/activities/RunningActivity.java

```diff
@@ -603,7 +648,7 @@ public void onError(Throwable t) {
 
     private void restartTango() {
         if (mParameterNode != null) mParameterNode.setPreferencesFromParameterServer();
-        updateSaveMapButton();
+        updateLoadAndSaveMapButtons();
```

Comment:
It looks like this function needs to be called in many places in this activity to keep the UI in sync with the Tango status. Shouldn't it be sufficient to only add it to `onTangoStatus`?

If Tango is restarted, the status should go from connected to disconnected and then to connected again, so the function onTangoStatus will be called in this cycle.